src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,22 @@
+import { INTERNAL_SERVER_ERROR, UNAUTHORIZED } from 'http-status-codes';
+
+import AuthService from '../services/authService';
+import { UserAuthModel } from '../models/authModel';
+
+const authService = new AuthService(UserAuthModel);
+const authMiddleware = async (req, res, next) => {
+    const token = req.header('authorization');
+    
+    try {
+        const userAuth = await authService.getUserByToken(token);
```

Comment:
I might not seeing something in here, but this middleware seems to missing some logic.
According to the task description 6.1 last 2 points - the middleware should check:
1. if the token is present - otherwise send Unauthorized Error
2. if the present token is valid - otherwise send Forbidden Error